pkg/volume/volume.go

```diff
@@ -132,11 +132,39 @@ type Attacher interface {
 	Attach() error
 }
 
```

Comment:
Comments on both should make it clear that the paths may not necessarily exist, just what the expected location is.